include/ghc-api-version.h

```diff
@@ -0,0 +1,10 @@
+#ifndef GHC_API_VERSION_H
+#define GHC_API_VERSION_H
+
+#ifdef GHC_LIB
+#define MIN_GHC_API_VERSION(x,y,z) MIN_VERSION_ghc_lib(x,y,z)
```

Comment:
Does this mean we can build hls with -fghc-lib now? That might solve the hlint ghc-lib mismatch issue @jneira

Reply:
I don't know about that, I have never tried to build ghcide with ghc-lib. Ping @cocreature